src/Cache.js

```diff
@@ -62,11 +62,13 @@ class Cache {
    * remove from cache after maxAge time has passed
    *
    * @param {*} key the key to remove
-   * @param {number} maxAge the time in milliseconds to wait before removing key
+   * @param {number} maxAge the time in milliseconds to wait before removing the key
+   * @param {Function} onExpiration a callback that is called after removing the key
    */
-  expireAfter(key: any, maxAge: number) {
+  expireAfter(key: any, maxAge: number, onExpiration: ?Function) {
     setTimeout(() => {
       this.remove(key);
+      if (onExpiration) onExpiration(key.key);
```

Comment:
To help with catching this kind of stuff I removed `prettier` and went back to the stricter ESLint-based style rules that I was using ... if u sync up with the latest master u should catch this now.